astro/src/content/quickstarts/quickstart-react-native.mdx

```diff
@@ -0,0 +1,213 @@
+
+## App Customization
+
+In this section, you'll turn your application into a trivial banking application with some styling.
+
+### Add Styling
+
+First, run the command below to install some libraries needed for theming.
+
+```shell
+npm install expo-image expo-constants react-native-currency-input
+```
+
+Instead of using CSS, React Native has its own concept of [stylesheets](https://reactnative.dev/docs/stylesheet). Create a file named `changebank.style.js` with the contents below to style your ChangeBank app.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/changebank.style.js"
+            lang="javascript"/>
+
+## Finish Setting up the App
+
+Replace the existing `App.js` to integrate `expo-auth-session`, add the ChangeBank template and stitch everything up.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/App.js"
+            lang="javascript"/>
+
+## Run the App
+
+Depending on where you want to test your app, follow these instructions.
+- For Android devices: you can either [connect a physical device](https://developer.android.com/studio/run/device) or install [Android Studio](https://developer.android.com/studio) to [set up an emulator](https://developer.android.com/studio/run/emulator).
+- For iOS devices: if you are running a Mac, you can install [Xcode](https://developer.apple.com/xcode/) to [set up an emulator or run on a physical device](https://developer.apple.com/documentation/xcode/running-your-app-in-simulator-or-on-a-device/).
+
+Now that you have a device connected or an emulator running, start up the React Native application from the root `my-react-native-app` directory using the command below.
+
+```shell
+npx expo start
+```
+
+After waiting a few moments, you should see a QR Code and a menu with some actions. Right below the QR Code, you'll see a message like this one _(the real address may vary)_.
+
+```
+› Metro waiting on exp://192.168.1.2:8081
+```
+
+To use [Expo Go](https://docs.expo.dev/get-started/expo-go/), a client for testing your apps on Android and iOS devices without building anything locally, you need to:
+
+* Copy that listening address (`exp://192.168.1.2:8081`).
+* Navigate to your FusionAuth instance on [localhost:9011](http://localhost:9011).
+* Browse to **Applications**.
+* Click on <IconButton icon="edit" color="blue" /> in your `ExampleReactNativeApp` to edit it.
+* Go to the **OAuth** tab.
+* Add that address to **Authorized redirect URLs** and append `/--/redirect` to it
+  * Example: for `exp://192.168.1.2:8081`, you'd need to add `exp://192.168.1.2:8081/--/redirect` as a redirect URL
+* Click on <IconButton icon="save" color="blue" /> to save your changes.
+
+Go back to the terminal with the Expo menu. To test on the connected or emulated Android device, press `a`. Otherwise, press `i` to run on the emulated iOS device.
+
```

Comment:
Not a big deal, but the formatted value for the Amount in US uses a comma for a separator.  In the change description, it uses a decimal.  That should be made consistent.  I think we mainly use decimal.
![Screenshot 2023-09-07 at 10 31 13 AM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/dd38df48-b8f8-4b7d-9553-aac5be0dde3f)

Reply:
Fixed, thanks. The author for the currency input is Brazilian and we have it the other way around :) Thanks